pkg/defaults/defaults.go

```diff
@@ -450,6 +452,15 @@ func stepConfigsForBuild(config *api.ReleaseBuildConfiguration, jobSpec *api.Job
 
 	for i := range config.Images {
 		image := &config.Images[i]
+		// If current image is operator bundle, add build step for its operator bundle source
+		if len(image.OperatorManifests) > 0 {
```

Comment:
nit: It's a plain string so let's compare it against an empty string for readability.